File: bioweb/src/java/entity/Entrada.java
package entity;

/**
 *
 * @author Crispancho
 */
public class Entrada {

    private String loguin;
    private String nombreProducto;
    private String nombreResponsable;
    private String nombreProveedor;
    private String nombreConductor;

    private int idEntrada;
    private int usuarioSistema;
    private int idProducto;
    private int responsable;
    private int proveedor;
    private String ruta;
    private int conductor;
    private int consecutivo;
    private String fechaHora;
    private float caneca;
    private float galon;
    private float kgRemisionado;
    private float kgPlanta;
    private float diferencia;
    private float productoMalo;
    private float perdidaPesoSegundoProceso;
    private float pesoNeto;
    private String observacion;
    private String numeroPlaca;

    /**
     * 
     * @param loguin
     * @param nombreProducto
     * @param nombreResponsable
     * @param nombreProveedor
     * @param nombreConductor
     * @param idEntrada
     * @param ruta
     * @param consecutivo
     * @param fechaHora
     * @param caneca
     * @param galon
     * @param kgRemisionado
     * @param kgPlanta
     * @param diferencia
     * @param productoMalo
     * @param perdidaPesoSegundoProceso
     * @param pesoNeto
     * @param observacion
     * @param numeroPlaca 
     * 
     * constructor espesifico para crear objetos de la consulta inner join
     */
    public Entrada(int idEntrada, String loguin, String nombreProducto, String nombreResponsable, String nombreProveedor, String ruta, String nombreConductor, String numeroPlaca, int consecutivo, String fechaHora, float caneca, float galon, float kgRemisionado, float kgPlanta, float diferencia, float productoMalo, float perdidaPesoSegundoProceso, float pesoNeto, String observacion) {
        
        this.idEntrada = idEntrada;
        this.loguin = loguin;
        this.nombreProducto = nombreProducto;
        this.nombreResponsable = nombreResponsable;
        this.nombreProveedor = nombreProveedor;
        this.nombreConductor = nombreConductor;
        this.ruta = ruta;
        this.consecutivo = consecutivo;
        this.fechaHora = fechaHora;
        this.caneca = caneca;
        this.galon = galon;
        this.kgRemisionado = kgRemisionado;
        this.kgPlanta = kgPlanta;
        this.diferencia = diferencia;
        this.productoMalo = productoMalo;
        this.perdidaPesoSegundoProceso = perdidaPesoSegundoProceso;
        this.pesoNeto = pesoNeto;
        this.observacion = observacion;
        this.numeroPlaca = numeroPlaca;
    }

    
    public Entrada() {
    }

   
    public String getLoguin() {
        return loguin;
    }

    public void setLoguin(String loguin) {
        this.loguin = loguin;
    }

    public String getNombreProducto() {
        return nombreProducto;
    }

    public void setNombreProducto(String nombreProducto) {
        this.nombreProducto = nombreProducto;
    }

    public String getNombreResponsable() {
        return nombreResponsable;
    }

    public void setNombreResponsable(String nombreResponsable) {
        this.nombreResponsable = nombreResponsable;
    }

    public String getNombreProveedor() {
        return nombreProveedor;
    }

    public void setNombreProveedor(String nombreProveedor) {
        this.nombreProveedor = nombreProveedor;
    }

    public String getNombreConductor() {
        return nombreConductor;
    }

    public void setNombreConductor(String nombreConductor) {
        this.nombreConductor = nombreConductor;
    }

    public int getIdEntrada() {
        return idEntrada;
    }

    public void setIdEntrada(int idEntrada) {
        this.idEntrada = idEntrada;
    }

    public int getUsuarioSistema() {
        return usuarioSistema;
    }

    public void setUsuarioSistema(int usuarioSistema) {
        this.usuarioSistema = usuarioSistema;
    }

    public int getIdProducto() {
        return idProducto;
    }

    public void setIdProducto(int idProducto) {
        this.idProducto = idProducto;
    }

    public int getResponsable() {
        return responsable;
    }

    public void setResponsable(int responsable) {
        this.responsable = responsable;
    }

    public int getProveedor() {
        return proveedor;
    }

    public void setProveedor(int proveedor) {
        this.proveedor = proveedor;
    }

    public String getRuta() {
        return ruta;
    }

    public void setRuta(String ruta) {
        this.ruta = ruta;
    }

    public int getConductor() {
        return conductor;
    }

    public void setConductor(int conductor) {
        this.conductor = conductor;
    }

    public int getConsecutivo() {
        return consecutivo;
    }

    public void setConsecutivo(int consecutivo) {
        this.consecutivo = consecutivo;
    }

    public String getFechaHora() {
        return fechaHora;
    }

    public void setFechaHora(String fechaHora) {
        this.fechaHora = fechaHora;
    }

    public float getCaneca() {
        return caneca;
    }

    public void setCaneca(float caneca) {
        this.caneca = caneca;
    }

    public float getGalon() {
        return galon;
    }

    public void setGalon(float galon) {
        this.galon = galon;
    }

    public float getKgRemisionado() {
        return kgRemisionado;
    }

    public void setKgRemisionado(float kgRemisionado) {
        this.kgRemisionado = kgRemisionado;
    }

    public float getKgPlanta() {
        return kgPlanta;
    }

    public void setKgPlanta(float kgPlanta) {
        this.kgPlanta = kgPlanta;
    }

    public float getDiferencia() {
        return diferencia;
    }

    public void setDiferencia(float diferencia) {
        this.diferencia = diferencia;
    }

    public float getProductoMalo() {
        return productoMalo;
    }

    public void setProductoMalo(float productoMalo) {
        this.productoMalo = productoMalo;
    }

    public float getPerdidaPesoSegundoProceso() {
        return perdidaPesoSegundoProceso;
    }

    public void setPerdidaPesoSegundoProceso(float perdidaPesoSegundoProceso) {
        this.perdidaPesoSegundoProceso = perdidaPesoSegundoProceso;
    }

    public float getPesoNeto() {
        return pesoNeto;
    }

    public void setPesoNeto(float pesoNeto) {
        this.pesoNeto = pesoNeto;
    }

    public String getObservacion() {
        return observacion;
    }

    public void setObservacion(String observacion) {
        this.observacion = observacion;
    }

    public String getNumeroPlaca() {
        return numeroPlaca;
    }

    public void setNumeroPlaca(String numeroPlaca) {
        this.numeroPlaca = numeroPlaca;
    }

    @Override
    public String toString() {
        return "Entrada{" + "loguin=" + loguin + ", nombreProducto=" + nombreProducto + ", nombreResponsable=" + nombreResponsable + ", nombreProveedor=" + nombreProveedor + ", nombreConductor=" + nombreConductor + ", idEntrada=" + idEntrada + ", usuarioSistema=" + usuarioSistema + ", idProducto=" + idProducto + ", responsable=" + responsable + ", proveedor=" + proveedor + ", ruta=" + ruta + ", conductor=" + conductor + ", consecutivo=" + consecutivo + ", fechaHora=" + fechaHora + ", caneca=" + caneca + ", galon=" + galon + ", kgRemisionado=" + kgRemisionado + ", kgPlanta=" + kgPlanta + ", diferencia=" + diferencia + ", productoMalo=" + productoMalo + ", perdidaPesoSegundoProceso=" + perdidaPesoSegundoProceso + ", pesoNeto=" + pesoNeto + ", observacion=" + observacion + ", numeroPlaca=" + numeroPlaca + '}';
    }

    
}

File: bioweb/entity/Entrada.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Entrada
 *
 * @author crispancho
 */
class Entrada {

    private $idEntrada;
    private $usuarioSistema;
    private $idProducto;
    private $responsable;
    private $proveedor;
    private $ruta;
    private $conductor;
    private $consecutivo;
    private $fechaHora;
    private $caneca;
    private $galon;
    private $kgRemisionado;
    private $kgPlanta;
    private $diferencia;
    private $productoMalo;
    private $perdidaPesoSegundoProceso;
    private $pesoNeto;
    private $observacion;
    private $numeroPlaca;
    
    
    function __construct() {
        
    }

    function getIdEntrada() {
        return $this->idEntrada;
    }

    function getUsuarioSistema() {
        return $this->usuarioSistema;
    }

    function getIdProducto() {
        return $this->idProducto;
    }

    function getResponsable() {
        return $this->responsable;
    }

    function getProveedor() {
        return $this->proveedor;
    }

    function getRuta() {
        return $this->ruta;
    }

    function getConductor() {
        return $this->conductor;
    }

    function getConsecutivo() {
        return $this->consecutivo;
    }

    function getFechaHora() {
        return $this->fechaHora;
    }

    function getCaneca() {
        return $this->caneca;
    }

    function getGalon() {
        return $this->galon;
    }

    function getKgRemisionado() {
        return $this->kgRemisionado;
    }

    function getKgPlanta() {
        return $this->kgPlanta;
    }

    function getDiferencia() {
        return $this->diferencia;
    }

    function getProductoMalo() {
        return $this->productoMalo;
    }

    function getPerdidaPesoSegundoProceso() {
        return $this->perdidaPesoSegundoProceso;
    }

    function getPesoNeto() {
        return $this->pesoNeto;
    }

    function getObservacion() {
        return $this->observacion;
    }

    function getNumeroPlaca() {
        return $this->numeroPlaca;
    }

    function setIdEntrada($idEntrada) {
        $this->idEntrada = $idEntrada;
    }

    function setUsuarioSistema($usuarioSistema) {
        $this->usuarioSistema = $usuarioSistema;
    }

    function setIdProducto($idProducto) {
        $this->idProducto = $idProducto;
    }

    function setResponsable($responsable) {
        $this->responsable = $responsable;
    }

    function setProveedor($proveedor) {
        $this->proveedor = $proveedor;
    }

    function setRuta($ruta) {
        $this->ruta = $ruta;
    }

    function setConductor($conductor) {
        $this->conductor = $conductor;
    }

    function setConsecutivo($consecutivo) {
        $this->consecutivo = $consecutivo;
    }

    function setFechaHora($fechaHora) {
        $this->fechaHora = $fechaHora;
    }

    function setCaneca($caneca) {
        $this->caneca = $caneca;
    }

    function setGalon($galon) {
        $this->galon = $galon;
    }

    function setKgRemisionado($kgRemisionado) {
        $this->kgRemisionado = $kgRemisionado;
    }

    function setKgPlanta($kgPlanta) {
        $this->kgPlanta = $kgPlanta;
    }

    function setDiferencia($diferencia) {
        $this->diferencia = $diferencia;
    }

    function setProductoMalo($productoMalo) {
        $this->productoMalo = $productoMalo;
    }

    function setPerdidaPesoSegundoProceso($perdidaPesoSegundoProceso) {
        $this->perdidaPesoSegundoProceso = $perdidaPesoSegundoProceso;
    }

    function setPesoNeto($pesoNeto) {
        $this->pesoNeto = $pesoNeto;
    }

    function setObservacion($observacion) {
        $this->observacion = $observacion;
    }

    function setNumeroPlaca($numeroPlaca) {
        $this->numeroPlaca = $numeroPlaca;
    }


}
